images/linux/scripts/installers/homebrew-doc.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+################################################################################
+##  File:  homebrew-doc.sh
+##  Desc:  Document the installation of Homebrew on Linux
+################################################################################
+
+# Source the helpers
+source $HELPER_SCRIPTS/document.sh
+
+# Document the installed version
+echo "Document the installed version"
+DocumentInstalledItem "$(cat /tmp/homebrew.txt)"
```

Comment:
Why we need a separate script for documentation?

Reply:
Because homebrew must be installed and run as `runner` but the file with meta information used for documentation is owned by `root` user.

Reply:
Is it possible to use the single script and just run a few command using the other user? Probably, it will be more convenient than split installation to a few parts because we don't use this approach for other tools

Reply:
https://github.com/actions/virtual-environments/pull/589/commits/cdb7f4b0cf28281c6253fe5e1d48876a3c451ac0